falco/values-k8saudit.yaml

```diff
@@ -1,11 +1,34 @@
+# -- Disable the drivers since we want to deplouy only the k8saudit plugin.
 driver:
   enabled: false
 
+# -- Disable the collectors, no syscall events to enrich with metadata.
 collectors:
   enabled: false
 
+# -- Deploy Falco as a deployment. One instance of Falco is enough. Anyway the number of replicas is configurabale.
 controller:
   kind: deployment
+  deployment:
+    # -- Number of replicas when installing Falco using a deployment. Change it if you really know what you are doing.
+    # For more info check the section on Plugins in the README.md file.
+    replicas: 1
+
+
+falcoctl:
+  artifact:
+    install:
+      # -- Enable the falcoctl tool as init container. It installs artifacts in the config.artifact.install.refs list.
+      enabled: true
+    follow:
+      # -- Disable the sidecar container. We do not support it yet for plugins. It is used only for rules feed such as Falco rules.
+      enabled: false
```

Comment:
We may want to enable this only for `k8saudit-rules`. Perhaps, just explaining that in the comment would be enough for now.